beacon-chain/rpc/service.go

```diff
@@ -212,6 +213,9 @@ func (s *Service) Start() {
 	}
 	s.grpcServer = grpc.NewServer(opts...)
 
+	secondsPerEpoch := params.BeaconConfig().SecondsPerSlot * params.BeaconConfig().SlotsPerEpoch
+	epochTicker := slotutil.GetSlotTicker(s.genesisTimeFetcher.GenesisTime(), secondsPerEpoch)
```

Comment:
Sanity check: Does this work pre-chainstart or before the genesis time is determined?